org.eclipse.jdt.ui.tests.refactoring/resources/InlineMethodWorkspace/TestCases/binary/.gitignore

```diff
@@ -0,0 +1 @@
+/classes/
```

Comment:
This should not be necessary and the following 3 files should not be part of the PR.

Reply:
This is noted with thanks and will address this accordingly.